Userland/Libraries/LibWeb/Bindings/MainThreadVM.cpp

```diff
@@ -422,8 +422,8 @@ ErrorOr<void> initialize_main_thread_vm(HTML::EventLoop::Type type)
         // 3. Let referencingScript be null.
         Optional<HTML::Script&> referencing_script;
 
-        // FIXME: 4. Let fetchOptions be the default classic script fetch options.
-        auto fetch_options = HTML::default_classic_script_fetch_options();
+        // FIXME: 4. Let originalFetchOptions be the default classic script fetch options.
```

Comment:
Is the FIXME still needed here?

Reply:
I wondered the same thing earlier - I couldn't work out why it was still in place. I'll remove it :)